### ISSUE TITLE
Set Canvas width/height correctly

### ISSUE THREAD

**Issue by takahirox** [mozilla/hubs#5465]
**Description**

Hubs Client seems to unexpectedly set CSS screen resolution (`screen.width, .height`), not Screen device resolution (CSS resolution * device pixel ratio), to Canvas width and height.

* `maxResolution` is `screen.width` and `.height` by default (unless they are set from the preferences)
* Imagine that browser is full screen (ie. mobile device)
* [`getRenderResolution()` returns `maxResolution` / pixel ratio](https://github.com/mozilla/hubs/blob/01d8e31dbdf1f300174f0fdc17dc9d5cb4f73580/src/react-components/room/useResizeViewport.js#L7-L24)
* The returned values are passed to `renderer.setSize()`
* [`renderer.setSize()` sets the passed size * pixel ratio to Canvas width and height](https://github.com/MozillaReality/three.js/blob/58551805d11cbc5510989950beb35d0141ca9a71/src/renderers/WebGLRenderer.js#L409-L410)

In short, `screen.width, .height` / pixel ratio * pixel ratio are set to Canvas width and height.

But for the best visual quality, Screen device resolution (`screen.width, .height` * pixel ratio) should be set to Canvas width and height.

This problem causes the blurriness on mobile devices.

**Expected behavior**

Screen device resolution are set to Canvas width and height (in full screen).

**Hardware** 
 - Device: Any
 - OS: Any
 - Browser: Any

**Additional context**

Currently the canvas size are smaller than it needs to be and the visual quality is worse. If we fix the problem the visual quality can improve but the performance can be affected.

We may need to consider smaller resolution for mobile devices.
